stix_shifter_modules/cybereason/stix_transmission/api_client.py

```diff
@@ -50,16 +52,18 @@ async def session_log_in(self):
         :return: str, cookie id
         """
         headers = {"Content-Type": "application/x-www-form-urlencoded"}
-        response_wrapper = await self.client.call_api(self.LOGIN_ENDPOINT, 'POST', headers=headers, data=self.auth)
-
-        return response_wrapper.response.cookies
+        async with aiohttp.ClientSession(headers=headers) as session:
```

Comment:
Isn't it still possible to use the self.client which is initiated from general [RestApiClientAsync](https://github.com/opencybersecurityalliance/stix-shifter/blob/release/5.0.x/stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py)? 
I am going to try one thing and if successful will suggest further here.

Reply:
@baulus ,
We have tried using "self.client," but we couldn't get cookies.
Let us know any possible way to get cookies using "self.client".

Reply:
I have added support in [RestApiClientAsync](https://github.com/opencybersecurityalliance/stix-shifter/pull/1310/files) to get and set cookies
You will get it once you change the PR target to [release/5.0.x](https://github.com/opencybersecurityalliance/stix-shifter/tree/release/5.0.x) and bring your branch up-to-date with release/5.0.x
for getting a cookie you need to use:
```
url = self.host+self.LOGIN_ENDPOINT
response = await self.client.call_api(url, 'POST', headers=headers, data=self.auth)
cookie = response.get_cookies(url)
```
for setting cookies just add `cookies=cookies` in `self.client.call_api` arguments. 
cookies var is dictionary, just like header

Reply:
Created a new PR for this changes and target to release/5.0.x branch.
https://github.com/opencybersecurityalliance/stix-shifter/pull/1313